src/Microsoft.AspNet.SignalR.JS/Microsoft.AspNet.SignalR.JS.csproj

```diff
@@ -21,16 +21,30 @@
 
   <ItemGroup>
     <JSCompile Include="jquery.signalr.*.js" />
+    <JSCompile Include="package.json" />
+    <JSCompile Include="LICENSE.md" />
+    <JSCompile Include="README.md" />
   </ItemGroup>
 
   <ItemGroup>
-    <GeneratedJavaScript Include="$(OutDir)jquery.signalR-$(PackageVersion).js" />
-    <GeneratedJavaScript Include="$(OutDir)jquery.signalR-$(PackageVersion).min.js" />
+    <GeneratedJavaScript Include="$(OutputPath)jquery.signalR-$(PackageVersion).js" />
+    <GeneratedJavaScript Include="$(OutputPath)jquery.signalR-$(PackageVersion).min.js" />
+    <GeneratedJavaScript Include="$(OutputPath)jquery.signalR.min.js" />
```

Comment:
We're now generating 2 more .js files?

Reply:
Are the versions embedded in them?

Reply:
a) we were already generating them, this just adds them to the Outputs for MSBuild to detect that they need to be rebuilt

b) these are for the tests and NPM package, both of which expect a versionless file path

c) yes, the versions are embedded in all of the JS files (even the ones with the version number in the file name)